package.json

```diff
@@ -39,6 +40,7 @@
   "devDependencies": {
     "chalk": "2.4.2",
     "chromedriver": "76.0.1",
+    "coveralls": "^3.0.6",
```

Comment:
Could you fix the version, like we do for the other devDependencies?

Reply:
yes, sure